devops/ansible/examples/girder-dev-environment/site.yml

```diff
@@ -70,3 +70,11 @@
       become: yes
       become_user: root
       when: ansible_os_family == "RedHat"
+
+    - name: Activate virtualenv on login
+      lineinfile:
+        dest: ~/.profile
+        line: 'source {{girder_virtualenv}}/bin/activate'
+        insertafter: 'EOF'
```

Comment:
We don't actually care where the line is inserted, so it may be best to omit this line, to allow `lineinfile` to apply its own best judgement (which defaults to 'EOF' anyway).